add_ia_covers/cover_updater.py

```diff
@@ -0,0 +1,30 @@
+import csv
```

Comment:
Let's add a description on this - that this is the bot that takes blank covers and adds a cover to them.